lib/registries/addon/components/page-link/component.ts

```diff
@@ -84,13 +84,13 @@ export default class PageLinkComponent extends Component {
     get pageIcon(): string {
         switch (this.pageState) {
         case PageState.Active:
-            return 'circle-o';
+            return 'dot-circle';
```

Comment:
Ideally this would be using `circle` but with a different prefix, but that would be a big pain to implement, and `dot-circle` gets the same idea across. `circle` in the default icon set is just a solid circle.